economy/work.go.tmpl

```diff
@@ -0,0 +1,28 @@
+{{/*
+	This command lets you work.
+	Usage: "-work"
+ 
+	Recommended trigger: work
+	Trigger type: Command
+ 
+	Credits:
+	Made by Jordi
+*/}}
+{{/* Configuration variables start */}}
+{{ $name := "coins" }} {{/* the name of the currency */}}
+{{ $min := 500 }} {{/* the minimum amount people should get for working*/}}
+{{ $max := 1000}} {{/* the maximum amount people should get for working*/}}
+{{ $cooldown := 20}} {{/* the cooldown before users can work again*/}}
+{{/* Configuration variables end */}}
+
+{{ $subt := randInt $min $max }}
```

Comment:
maybe check if min < max so it wont give error

Reply:
will do